pkg/cmd/cli/install/install.go

```diff
@@ -278,6 +278,13 @@ func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
 		return errors.Wrap(err, errorMsg)
 	}
 
+	if o.UseRestic {
+		fmt.Println("Waiting for Velero restic daemonset to be ready.")
+		if _, err = install.DaemonSetIsReady(factory, o.Namespace); err != nil {
+			return errors.Wrap(err, errorMsg)
+		}
+	}
+
```

Comment:
Isn't it duplicate? https://github.com/vmware-tanzu/velero/blob/002bc56ad360c6b3604bc7c9508b7252ebe6f53e/pkg/cmd/cli/install/install.go#L294-L299